pandas/tests/io/json/test_pandas.py

```diff
@@ -1018,13 +1018,19 @@ def test_sparse(self):
         df = pd.DataFrame(np.random.randn(10, 4))
         df.loc[:8] = np.nan
 
-        sdf = df.to_sparse()
+        # GH 26557: DEPR
+        with tm.assert_produces_warning(FutureWarning,
```

Comment:
I think here, you can also a filterwarning instead (since the test already has a filterwarning for the main Sparse deprecation as well)

Reply:
Will check on that, but we still have to come to an unanimous decision whether to keep the double deprecation warning for `Series/DataFrame.to_sparse`. None of this will be needed if we deicide to stick with only 1 warning for the main deprecation.

Reply:
> Will check on that, but we still have to come to an unanimous decision whether to keep the double deprecation warning for `Series/DataFrame.to_sparse`. None of this will be needed if we deicide to stick with only 1 warning for the main deprecation.

We should not keep the double warning at all.